ophyd/areadetector/filestore_mixins.py

```diff
@@ -124,7 +124,7 @@ def generate_datum(self, key, timestamp):
     def describe(self):
         # One object has been 'described' once, no new keys can be added
         # during this stage/unstage cycle.
-        self._locked_key_list = self._staged
+        self._locked_key_list = (self._staged == Staged.yes)
```

Comment:
not to do in this PR, but did we change our mind that this was too clever or not?